desktop/app.js

```diff
@@ -3,9 +3,14 @@ var Menu = require( 'menu' );
 var BrowserWindow = require( 'browser-window' );	// Module to create native browser window.
 var path = require('path')
 
-module.exports = function main( url ) {
+require( 'module' ).globalPaths.push( path.resolve( path.join( __dirname ) ) );
+
+module.exports = function main() {
+	var url = 'file://' + path.join( __dirname, '..', 'dist', 'index.html' );
```

Comment:
`path.join` will indeed use `\` on Windows, but Chromium handles that just fine. It changes the separator to `/`, and also adds the third slash to `file://`. There's no need for us to detect the OS and replace `\` with `/`.